crates/driver/src/infra/config/file/mod.rs

```diff
@@ -260,6 +268,15 @@ struct SolverConfig {
     /// Maximum HTTP response size the driver will accept in bytes.
     #[serde(default = "default_response_size_limit_max_bytes")]
     response_size_limit_max_bytes: usize,
+
+    /// Which tokens are explicitly supported or unsupported by the solver.
+    #[serde(default)]
+    token_supported: HashMap<eth::H160, bool>,
+
+    /// Whether or not the solver opted into detecting unsupported
+    /// tokens with `trace_callMany` based simulation.
+    #[serde(default)]
+    enable_simulation_bad_token_detection: bool,
```

Comment:
nit: why not just `enable_bad_token_detection`

Reply:
there will be different strategies for bad token detection so we'd have to be more specific than that